_posts/2021-07-25-personal-allowance.md

```diff
@@ -0,0 +1,1319 @@
+<div id="code_block_47" style="display: none;">
+  <pre>
+    <code>
+#Calc
+mtr_two_kids = graphs.mtr_chart(reform_list, names=reform_names, variables=["household_net_income", "tax", "benefits"],
+                situation_function=couple_two_children)
+    </code>
+  </pre>
+</div>
+
+<script>
+function show_code_47() {
+  var x = document.getElementById("code_block_47");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+}
+</script>
+
+
+<button onclick="show_code_48()">Click to show code</button>
+<div id="code_block_48" style="display: none;">
+  <pre>
+    <code>
+#Graph
+format_fig(mtr_two_kids.update_layout(
+    title_text='Marginal tax rate for a single person when replacing the personal allowance with a UBI'))
+    </code>
+  </pre>
+</div>
+
+<script>
+function show_code_48() {
+  var x = document.getElementById("code_block_48");
+  if (x.style.display === "none") {
+    x.style.display = "block";
+  } else {
+    x.style.display = "none";
+  }
+}
+</script>
+
+### Conclusion
+
+My research aligns with other analyses showing the progressivity of replacing the personal allowance with UBI. In 2019, the New Economics Foundation [modeled](https://neweconomics.org/2019/03/nothing-personal) replacing the personal allowance with a budget-neutral Weekly National Allowance paid to adults and an increase in the child benefit, making the payments non-taxable but included in means tests. They found progressive results, stating it would “[lift] 200,000 families out of poverty and shift the £8bn currently spent on tax allowances for the 35% highest income families to the remaining 65% of families”. Compass also [modeled](https://www.compassonline.org.uk/wp-content/uploads/2019/03/Compass_BasicIncomeForAll_2019.pdf) replacing the personal allowance with a budget-neutral flat payment of £25 a week for children and adults of working age, as well as a flat rate citizen’s pension of £164.35. They too found progressive results, with the Gini coefficient decreasing from 0.337 to 0.365. Most recently, [my colleagues found](https://www.ubicenter.org/progressive-adjustments-lib-dem-working-group) that the policy in the Liberal Democrats discussion paper could be made more progressive by repealing all, rather than part, of the personal allowance.
+
+Beyond these other papers, I've found that the impacts occur roughly linearly on the margin, meaning that the policy concept can operate incrementally. Furthermore, the policy disproportionately benefits children, and separate studies show that this will in turn improve educational and health outcomes. The impacts are largely but not uniformly beneficial to low income households, though losses are small.
+
+Given these findings, UBI proponents in the UK should take a serious look at whether keeping the personal allowance is worthwhile, provided the benefits of the UBI that could be funded through eliminating it.
```

Comment:
Feel free to adjust, but in general looking to reduce normativity/prescriptiveness (also considering that our audience extends beyond UBI proponents)
```suggestion
Reforms that broaden the tax base are often regressive on their own, but how the revenue is spent matters. Repealing the personal allowance is one such instance: using the revenue to fund a UBI switches the distributional sign from regressive to highly progressive.
```